.github/actions/alicenet-config/action.yml

```
@@ -3,5 +3,14 @@ description: "Configure the environment for alicenet"
 runs:
   using: "composite"
   steps:
-    - uses: ./.github/actions/golang-cache
+    - uses: ./.github/actions/npm-cache
     - uses: ./.github/actions/geth-eth-cache
+    - uses: ./.github/actions/golang-cache
+    - name: npm Compile
+      working-directory: ./bridge
+      shell: bash
+      run: npm run compile
```

```suggestion
    - run: npm run compile
      working-directory: ./bridge
      shell: bash
    - run: npm ci
      working-directory: ./bridge
      shell: bash
```

Should be all that's needed to get the tests running.